src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1219,6 +1353,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 // cursorTimer timer, now stopped, is destroyed.
             }
 
+            _autoScrollTimer.Stop();

```

Comment:
Technically this should be cleaned up the same way the other ones are, with swapping null for a local one and stopping the local one, but I think we changed the rest of the method over after this PR was authored. I can file a follow-up issue to polish that line.

Reply:
Then `_autoScrollTimer` would have to be nullable, though I don't see the point of doing so.